lib/codec.js

```diff
@@ -18,6 +18,7 @@
   exports.encodeBytes = f.lift(doEncodeBytes, _.identity);
   exports.encodeString = f.lift(doEncodeString, _.identity);
   exports.encodeSignedInt = f.lift(doEncodeSignedInt, _.identity);
+  exports.encodeShort = f.lift(doEncodeShort, _.identity);
```

Comment:
Is there a need for this? As far as I know, the protocol does not currently encode any shorts.